docs/SQL/development/visual-editor/interactive-development/interactive-development.md

```diff
@@ -12,6 +12,14 @@ tags:
   - data
 ---
 
+You can leverage Data Copilot for SQL by building your Model and adding individual Gems, and then check how the process is going by interactively running the model. Other interactive development features include:
+
+- Data Explorer
+- Interim
```

Comment:
The "Data Explorer" feature is essentially the umbrella term for the features: interim, sort, filter, download. I'd suggest:

You can leverage Data Copilot for SQL by building your Model and adding individual Gems, and then check how the process is going by interactively running the model. Use the Data Explorer to dig into your interim data sample, with the ability to sort, filter, and download your sample data.

Reply:
Got it.